presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -332,6 +333,19 @@ public FeaturesConfig setGroupedExecutionForAggregationEnabled(boolean groupedEx
         return this;
     }
 
+    public boolean isGroupedExecutionForAllEligibleTableScansEnabled()
+    {
+        return groupedExecutionForAllEligibleTableScansEnabled;
+    }
+
+    @Config("grouped-execution-for-all-eligible-table-scans-enabled")
```

Comment:
any thought about the name @rschlussel ? -- do we need "all" from grammar perspective?  😃

Reply:
i don't think we need 'all'.  it does emphasize that we'll do grouped execution even if there's no aggregation/join on top, but I still think it's better without 'all' to make it shorter.